Testing/UnitTests/xASL_ut_function_xASL_bids_BIDS2Legacy.m

```diff
@@ -44,7 +44,7 @@
 if ~exist(fullfile(droTestPatient,'derivatives'),'dir')
     testCondition = false; % Test failed
 end
-if ~exist(fullfile(droTestPatient,'derivatives','ExploreASL','DataPar.json'),'file')
+if ~exist(fullfile(droTestPatient,'derivatives','ExploreASL','dataPar.json'),'file')
```

Comment:
I guess here it strictly cannot be `dataPar_somethingelse.json`?

Reply:
Not really, since the default case creates a `dataPar.json `